plugins/inputs/smart/README.md

```diff
@@ -0,0 +1,103 @@
+# Telegraf S.M.A.R.T. plugin
+
+Get metrics using the command line utility `smartctl` for S.M.A.R.T. (Self-Monitoring, Analysis and Reporting Technology) storage devices. SMART is a monitoring system included in computer hard disk drives (HDDs) and solid-state drives (SSDs)[1] that detects and reports on various indicators of drive reliability, with the intent of enabling the anticipation of hardware failures.
+See smartmontools (https://www.smartmontools.org/).
+
+If no devices are specified, the plugin will scan for SMART devices via the following command:
+
+```
+smartctl --scan
+```
+
+Metrics will be reported from the following `smartctl` command:
+
+```
+smartctl --info --attributes --health -n <nocheck> --format=brief <device>
+```
+
+This plugin supports _smartmontools_ version 5.41 and above, but v. 5.41 and v. 5.42
+might require setting `nocheck`, see the comment in the sample configuration.
+
+To enable SMART on a storage device run:
+
+```
+smartctl -s on <device>
+```
+
+## Measurements
+
+- smart_device:
+
+    * Tags:
+      - `capacity`
+      - `device`
+      - `device_model`
+      - `enabled`
+      - `health`
+      - `serial_no`
+    * Fields:
+      - `exit_status`
+
+- smart_attribute:
+
+    * Tags:
+      - `device`
+      - `fail`
+      - `flags`
+      - `id`
+      - `name`
+    * Fields:
+      - `exit_status`
```

Comment:
I kept this and added some info in the README. It consists of a bit pattern that can be useful to find drives that are in some way failing or starting to fail.